cmd/context.go

```diff
@@ -109,6 +109,27 @@ var clearContextCmd = &cobra.Command{
 	},
 }
 
+var loginContextCmd = withLocalFlags(&cobra.Command{
+	Use:   "login <context-name>",
+	Args:  cobra.RangeArgs(0, 1),
+	Short: "Login and set cookie for the named context",
+	Long:  "Login and set cookie for the named context",
```

Comment:
Maybe this `long` description can be a bit longer :) Should probably include that this can be used to log in through the proxy on QSEfW.